types/rc-slider/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for rc-slider 8.1
```

Comment:
`dotStyle` and `activeDotStyle` are included in 8.2 only (not in 8.1).

Reply:
Corrected in 939d165